build/Builder.js

```diff
@@ -4,22 +4,22 @@ const sass = require('sass');
 const fs = require('fs');
 const path = require('path');
 
-class Builder{
+class Builder {
 	vars = {};
 
-	setVariable(name, value){
+	setVariable(name, value) {
 		this.vars[name] = value;
 		return this; //allow chaining
 	}
 
-	getVariables(){
-		const varString = Object.values(this.vars).map((k, v) => `$${k}: ${v};` ).join("\n");
+	getVariables() {
+		const varString = Object.entries(this.vars).map(arr => `$${arr[0]}: ${arr[1]};`).join("\n");
 		console.log(`Vars: ${varString}`);
 		return varString;
 	}
 
-	build(src, dest){
-		src = path.resolve(src);
+	build(src, dest) {
+		//src = path.resolve(src);
```

Comment:
Was this meant to be commented out or am I missing something here.

Reply:
Yes but no - it was commented out as a different fix (#33 ) but then I needed to do more work that depended on it...